es/refresher-linear-algebra.md

```diff
@@ -289,32 +288,31 @@ Notando Λ=diag(λ1,...,λn), se tiene que:
 
 **49. Gradient ― Let f:Rm×n→R be a function and A∈Rm×n be a matrix. The gradient of f with respect to A is a m×n matrix, noted ∇Af(A), such that:**
 
-&#10230; Gradiente ― Sea f:Rm×n→R una función y A∈Rm×n una matriz. El gradiente de f con respecto a A es una matriz de m×n, notando que ∇Af(A), tal que:
+&#10230; Gradiente ― Sea f:Rm×n→R una función y A∈Rm×n una matriz. El gradiente de f con respecto a A es una matriz de m×n, denotada por ∇Af(A), tal que:
 
 <br>
 
 **50. Remark: the gradient of f is only defined when f is a function that returns a scalar.**
 
-&#10230; Nota: el gradiente de f esta solo definido cuando f es una función cuyo resultado es un escalar.
+&#10230; Observación: el gradiente de f se define únicamente cuando f es una función que retorna un escalar.
 
 <br>
 
 **51. Hessian ― Let f:Rn→R be a function and x∈Rn be a vector. The hessian of f with respect to x is a n×n symmetric matrix, noted ∇2xf(x), such that:**
 
-&#10230; Matriz Hessiana ― Sea f:Rn→R una función y x∈Rn un vector. La matriz hessiana o hessiano de f con recpecto a x
-es una matriz simétrica de n×n, para ∇2xf(x), tal que:
+&#10230; Hessiana ― Sea f:Rn→R una función y x∈Rn un vector. La hessiana de f con recpecto a x es una matriz simétrica n×n, denotada ∇2xf(x), tal que:
 
 <br>
 
 **52. Remark: the hessian of f is only defined when f is a function that returns a scalar**
 
-&#10230; Nota: la matriz hessiana de f solo esta definida cuando f es una función que devuelve un escalar
+&#10230; Observación: la matriz hessiana de f se define únicamente cuando f es una función que retorna un escalar
```

Comment:
Haha. I suggested changing "retorna" to "devuelve". In Spain sounds really weird. But if for you it is the other way around you can keep it as it is.